website/markdown/docs/links.mdx

```diff
@@ -26,11 +26,13 @@ import DocTitle from './components/doc-title'
 
 - [Edit in Xcode](/docs/commands/edit/)
 - [Generate projects](/docs/commands/generate/)
+- [Sign your apps](/docs/commands/signing/)
```

Comment:
@fortmarek since we don't have a lot of signing the documentation, I moved the page under the commands section.